proxy/build.rs

```diff
@@ -0,0 +1,32 @@
+use std::env;
+use std::path::PathBuf;
+
+fn main() {
+    // Tell cargo to look for shared libraries in the specified directory
+    println!("cargo:rustc-link-search=/usr/lib");
+
+    // Tell cargo to tell rustc to link the system bzip2
+    // shared library.
+    println!("cargo:rustc-link-lib=qubesdb");
+
+    // The bindgen::Builder is the main entry point
+    // to bindgen, and lets you build up options for
+    // the resulting bindings.
+    let bindings = bindgen::Builder::default()
+        // The input header we would like to generate
+        // bindings for.
+        .header("wrapper.h")
+        // Tell cargo to invalidate the built crate whenever any of the
```

Comment:
Yes, in that:

https://github.com/freedomofpress/securedrop-client/blob/4853d4bc949cf77718b92d2399eed969ff7e6fd6/proxy/wrapper.h#L1

Let's chat if I'm not answering your question; I'm not sure I understand what you're asking.

Reply:
I was asking in respect to "Tell cargo to invalidate the built crate whenever any of the included header files changed." and whether bindgen was smart enough to follow the include of qubesdb-client and invalidate the cache. It was mostly theoretical, because our package builds won't be cached.